Source/Core/VideoCommon/ShaderCache.cpp

```diff
@@ -627,6 +617,21 @@ static GXPipelineUid ApplyDriverBugs(const GXPipelineUid& in)
     ps->ztest = EmulatedZ::Early;
   }
 
+  // We can use framebuffer fetch to emulate logic ops in the fragment shader.
```

Comment:
This comment seems wrong?  At least it says we use framebuffer fetch but then the below if check checks if framebuffer fetch is disabled.

Reply:
I added "So don't use the worse blend approximation." to the comment, does that help?

Reply:
"If framebuffer fetch is available, we can emulate logic ops in the fragment shader and don't need the below blend approximation."

Reply:
Didn't notice this earlier, applied